### PR TITLE
cmd/uplink: update CLI format

### PR DESCRIPTION
**Pull request by montyanderson**

### CODE REVIEW

File: cmd/uplink/cmd/setup.go [hunk 1]
```diff
@@ -157,9 +157,7 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 	fmt.Println(`
 Your Uplink CLI is configured and ready to use!
 
-Some things to try next:
-
-* See https://documentation.tardigrade.io/api-reference/uplink-cli for some example commands`)
+* See http://documentation.tardigrade.io/api-reference/uplink-cli for some example commands`)
```
Comment (kaloyan-raev) [inline cmd/uplink/cmd/setup.go]: Please, keep this URL https.

File: cmd/internal/wizard/wizard.go [hunk 2]
```diff
@@ -143,7 +143,10 @@ func PromptForAPIKey() (string, error) {
 
 // PromptForEncryptionPassphrase handles user input for an encryption passphrase to be used with wizards
 func PromptForEncryptionPassphrase() (string, error) {
-	_, err := fmt.Print("Enter your encryption passphrase: ")
+	_, err := fmt.Print(`
+Data is encrypted on the network, with an encryption passphrase
```
Comment (kaloyan-raev) [inline cmd/internal/wizard/wizard.go]: It looks like an extra empty line is printed. This line should start at the end of the previous one.

Comment (montyanderson) [reply cmd/internal/wizard/wizard.go]: The CLI seemed weird to me without a line break but your way is more authentic to the original mockup.

File: cmd/uplink/cmd/setup.go (hunk 1 above)
Comment (montyanderson) [reply cmd/uplink/cmd/setup.go]: You're right. I just copied what is written in the mockup (http).

Comment (kaloyan-raev) [reply cmd/uplink/cmd/setup.go]: The mockup was based on an older state of the setup wizard. We changed the URL to https during the bakeoff week.

Comment (mniewrzal) [reply cmd/uplink/cmd/setup.go]: still no change to `https`

Comment (stefanbenten) [reply cmd/uplink/cmd/setup.go]: Done

File: cmd/internal/wizard/wizard.go (hunk 2 above)
Comment (stefanbenten) [reply cmd/internal/wizard/wizard.go]: Done

**Review by stefanbenten — APPROVED** [review]

**Review by egonelbre — APPROVED** [review]

**Pull request merged**